loleaflet/src/control/Control.LokDialog.js

```diff
@@ -1193,6 +1193,9 @@ L.Control.LokDialog = L.Control.extend({
 		L.DomEvent.on(canvas, 'mousedown mouseup', function(e) {
 			L.DomEvent.stop(e);
 
+			if (this._map.uiManager.isUIBlocked())
```

Comment:
Do we need this for the keydown/keyup events too, or do we already block typing into the document ?

Reply:
good catch, I blocked only the UI (dialogs, sidebar, etc), not document editing. so we need this for typing. but probably also for mouse events in the document

Reply:
Yes, sounds good.